### ISSUE TITLE
media-query not added.

### ISSUE THREAD

**Issue by ItsAJ1005** [ItsAJ1005/web-application-using-Animations#1]
This site is not fully supported by mobile devices. Media queries should be added for better accessibility through all devices.
